website/cue/reference/cli.cue

```diff
@@ -162,6 +162,13 @@ cli: {
 			type:        "integer"
 			env_var:     "VECTOR_THREADS"
 		}
+		"internal-log-rate-limit": {
+			_short:      "i"
```

Comment:
I don't _love_ `-i` (and not sure it's customizable), but I can't think of anything better so 🤷